sdk/maps/Azure.Maps.Search/src/Models/GeoJsonLineStringData.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Maps.Search.Models
+{
+    internal partial class GeoJsonLineStringData
```

Comment:
We don't need to add `CodeGenModel` here? And our Generated `GeoJsonLineStringData` is automatically changed to `internal`?

Reply:
Correct, we don't need to add CodeGenModel. It's changed.